src/pages/[...slug].tsx

```diff
@@ -7,17 +7,24 @@ import {
 import { drupalClient } from '@/utils/drupalClient'
 import { Node, nodeMeta } from '@/components/node'
 import { NodeTypes } from '@/types/node'
+import { getGlobalElements } from '@/lib/context/getGlobalElements'
+import Layout from '@/components/layout'
 
 interface NodeProps {
   node: NodeTypes
   additionalNode: NodeTypes
 }
 
 /** This passes any retrieved node to the generalized Node component. */
-export default function NodePage({ node, additionalNode }: NodeTypes) {
+export default function NodePage({ node, additionalNode, props }: NodeTypes) {
   if (!node) return null
+  if (!props) return null
```

Comment:
Why the change from `node` to `props` here? Is the `node` check no longer relevant?

Reply:
Props seems to be the convention used to access the data from globalElements, I can probably destructure props within the globalElements file but we would probably want to call it something like banners since we already have 'node' serving a different purpose in this file. I agree it's a little bit confusing

Reply:
That makes sense re: `props`, but is it no longer relevant if `node` is empty?

Reply:
I can't read.